src/main/java/seedu/kitchenhelper/notification/IngredientNotification.java

```diff
@@ -65,8 +65,8 @@ public Date stringToDate(String date1) {
 
     public void addToNotification(ArrayList<Ingredient> ingredientArrayList) {
         for (Ingredient i : ingredientArrayList) {
-            notification += i.getIngredientName() + "|" + i.getQuantity() + "|"
-                    + i.getPrice() + "|" + i.getExpiryDate() + "\n";;
+            notification += "Ingredient name : " + i.getIngredientName() + " | " + i.getQuantity() + " portion(s) | $"
```

Comment:
![image](https://user-images.githubusercontent.com/32284166/78218658-bcfea700-74f0-11ea-8dd6-affa898d9011.png)

As seen from the image above, just a suggestion but perhaps you can put like spaces in between to split the expiring ingredients, low in stock ingredients and expired ingredients? for better user readability?

Reply:
Good job on the changes made but I believe there are some typos made, should be "These ingredients are currently low in stock" and "These ingredients have expired"

Reply:
Updated based on suggestions.